### PR TITLE
build.ps1: Update the msbuild installer call

### PR DESCRIPTION
**Pull request by tristanlabelle**
Accounts for changes in https://github.com/apple/swift-installer-scripts/pull/224

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -1405,64 +1406,36 @@ function Build-DocC() {
...
-  Build-WiXProject bundle\installer.wixproj -Arch $HostArch -Bundle -Properties @{
-    MSI_LOCATION = "$($HostArch.BinaryCache)\msi\";
+    Copy-File "$($HostArch.BinaryCache)\installer-scripts\Release\$($HostArch.VSName)\*.msi" $Stage
+    Copy-File "$($HostArch.BinaryCache)\installer-scripts\Release\$($HostArch.VSName)\installer.exe" $Stage
   }
 }
 
 #-------------------------------------------------------------------
 
+Build-Installer
```
Comment (tristanlabelle) [inline build.ps1:1431]: You didn't see this

File: build.ps1 [hunk 2]
```diff
@@ -597,22 +597,23 @@ function Build-WiXProject() {
 
   $Properties = $Properties.Clone()
   TryAdd-KeyValue $Properties Configuration Release
-  TryAdd-KeyValue $Properties BaseOutputPath "$($Arch.BinaryCache)\msi\"
+  TryAdd-KeyValue $Properties BaseOutputPath "$($Arch.BinaryCache)\installer-scripts\"
```
Comment (compnerd) [inline build.ps1:600]: I really like this!  Or even `installer` perhaps?

File: build.ps1 [hunk 3]
```diff
@@ -597,22 +597,23 @@ function Build-WiXProject() {
...
   TryAdd-KeyValue $Properties ProductArchitecture $ArchName
   TryAdd-KeyValue $Properties ProductVersion $ProductVersionArg
 
   $MSBuildArgs = @("$SourceCache\swift-installer-scripts\platforms\Windows\$FileName")
   $MSBuildArgs += "-noLogo"
   $MSBuildArgs += "-restore"
+  $MSBuildArgs += "-maxCpuCount"
```
Comment (compnerd) [inline build.ps1:607]: Oh, this is a good idea!

File: build.ps1 [hunk 4]
```diff
@@ -597,22 +597,23 @@ function Build-WiXProject() {
...
   foreach ($Property in $Properties.GetEnumerator()) {
     if ($Property.Value.Contains(" ")) {
       $MSBuildArgs += "-p:$($Property.Key)=$($Property.Value.Replace('\', '\\'))"
     } else {
       $MSBuildArgs += "-p:$($Property.Key)=$($Property.Value)"
     }
   }
-  $MSBuildArgs += "-bl:$($Arch.BinaryCache)\msi\$ArchName-$([System.IO.Path]::GetFileNameWithoutExtension($FileName)).binlog"
-  $MSBuildArgs += "-ds:False"
+  $MSBuildArgs += "-binaryLogger:$($Arch.BinaryCache)\msi\$ArchName-$([System.IO.Path]::GetFileNameWithoutExtension($FileName)).binlog"
```
Comment (compnerd) [inline build.ps1:615]: 🙄 fine, have your long form names.

File: build.ps1 (hunk 2 above)
Comment (tristanlabelle) [reply build.ps1:600]: Sure!

Comment (tristanlabelle) [reply build.ps1]: Updated in latest

**Review by compnerd — APPROVED** [review]

**Pull request merged**